geode-core/src/main/java/org/apache/geode/internal/cache/DistributedClearOperation.java

```diff
@@ -211,7 +215,7 @@ protected boolean operateOnRegion(CacheEvent event, ClusterDistributionManager d
       switch (this.clearOp) {
         case OP_CLEAR:
           region.clearRegionLocally((RegionEventImpl) event, false, this.rvv);
```

Comment:
why you did not change this line too?

Reply:
It's a change suggested by IntelliJ, I don't know why it didn't suggest this line. I will change it

Reply:
Oh, this cast is necessary, the event is of type `CacheEvent` when passed in.